packages/components/pie-radio-group/src/radio-group.scss

```diff
@@ -1 +1,54 @@
 @use '@justeattakeaway/pie-css/scss' as p;
+
+.c-radioGroup {
+    --radio-group-offset: var(--dt-spacing-c);
+    --radio-group-offset--inline: var(--dt-spacing-e);
+    --radio-group-label-offset: var(--dt-spacing-a);
+
+    margin: 0;
+    padding: 0;
+    border: 0;
+    min-width: 0;
+
+    ::slotted(pie-radio) {
+        display: flex;
+    }
+
+    ::slotted(pie-form-label) {
+        display: block;
+        padding-block-end: var(--radio-group-label-offset);
+    }
+
+    &:not(.c-radioGroup--inline) {
+        ::slotted(pie-radio:not(:last-child)) {
+            margin-block-end: var(--radio-group-offset);
+        }
+    }
+}
+
+.c-radioGroup--inline {
+    display: flex;
+
+    align-items: flex-start;
+    flex-wrap: wrap;
+}
+
+.c-radioGroup.c-radioGroup--inline {
+    margin: calc(-1 * var(--radio-group-offset) / 2) calc(-1 * var(--radio-group-offset--inline) / 2);
```

Comment:
Why was it necessary to have negative margin?

Reply:
i referenced the same styles of the checkbox group for consistency. 
maybe @dandel10n can help with that?

Reply:
i simplified the styles and will see if we ca do the same thing for checkbox group